index.js

```diff
@@ -45,53 +47,52 @@ Codec.prototype.decodeValue = function (value, opts) {
 }
 
 Codec.prototype.encodeBatch = function (ops, opts) {
-  var self = this
```

Comment:
While we're at it, refactor to classes too?

Reply:
I like classes for my own stuff but less for modules, because they force everyone to use `new()` and classes. Users can't do:

```js
class Module {

}

function Userland () {
  // Error
  Module.call(this)
}

Object.setPrototypeOf(Userland.prototype, Module.prototype)

const userland = new Userland()
```

Reply:
I see, I haven't had any of these issues yet, but my use cases are likely limited. 👍  then